src/org/omegat/core/data/RealProject.java

```diff
@@ -279,9 +279,15 @@ public void createProject() {
             createDirectory(config.getDictRoot(), OConsts.DEFAULT_DICT);
             createDirectory(config.getTargetRoot(), OConsts.DEFAULT_TARGET);
             //createDirectory(m_config.getTMOtherLangRoot(), OConsts.DEFAULT_OTHERLANG);
-
             saveProjectProperties();
 
+            File glossaries = config.getWritableGlossaryFile().getAsFile();
+            if (glossaries.createNewFile()) {
+                try (BufferedWriter writer = Files.newBufferedWriter(glossaries.toPath())) {
+                    writer.write("# Glossaries with tab separation -*- coding: utf-8 -*-");
+                }
+            }
+
```

Comment:
This should probably be abstracted out to something like `GlossaryReaderTSV#createEmpty`.